uiowa/uiowa
# Adjust role mapping configuration

Resolves https://github.com/uiowa/uiowa/issues/5862

This PR starts to ignore uiowa_auth.settings:role_mappings so that add/edits can be done directly in the PROD active db. Because of this, the sites that split config for this no longer need it. An update hook is run to implement the ignore before the config splits are updated preserving any custom mappings. A drush command is established that can be run on all sites using `blt ume` to query sites that have custom mappings in addition to the defaults.

# To Test
Setup your local.blt.yml file with the following sites along with a few other sites (e.g. default, gis.sites.uiowa.edu, admissions.uiowa.edu, education.uiowa.edu)

- default
- admissions.uiowa.edu
- brand.uiowa.edu
- connect.studentlife.uiowa.edu
- education.uiowa.edu
- sitenow.uiowa.edu
- sppa.uiowa.edu
- webcommunity.sites.uiowa.edu

```
ddev blt dsa && ddev blt ume uicore-crm
```

- Confirm custom role mappings are still intact for brand, connect.studentlife, sitenow, and sppa

You can also login to the mappings (admin/config/people/hawkid)
